eslint.config.js

```diff
@@ -0,0 +1,11 @@
+
+const globals = require("globals");
+
+module.exports = [{
+  languageOptions: {
+    globals: {
+      ...globals.jquery,
+      ...globals.node,
+    },
+  },
+}];
```

Comment:
We should extend prettier to prevent rule collision issues:

```suggestion


module.exports = [{
  languageOptions: {
    globals: {
      ...globals.jquery,
      ...globals.node,
    },
  },
  plugins: ["prettier"],
  extends: ["plugin:prettier/recommended"],
}];
```

* [Prettier on integrating with a linter](https://prettier.io/docs/en/integrating-with-linters.html)
* [what and why](https://github.com/prettier/eslint-config-prettier?tab=readme-ov-file#what-and-why)